src/system-health/health_checker/sysmonitor.py

```diff
@@ -1,30 +1,36 @@
 #!/usr/bin/python3
 
+import glob
+import multiprocessing
 import os
 import sys
 import time
-import glob
-import multiprocessing
+from copy import copy
 from datetime import datetime
-from swsscommon import swsscommon
+
 from sonic_py_common.logger import Logger
-from . import utils
 from sonic_py_common.task_base import ProcessTaskBase
 from sonic_py_common import device_info
+from swsscommon import swsscommon
+
+from . import utils
 from .config import Config
 import signal
 import jinja2
 
 SYSLOG_IDENTIFIER = "system#monitor"
 REDIS_TIMEOUT_MS = 0
-system_allsrv_state = "DOWN"
-spl_srv_list = ['database-chassis', 'gbsyncd']
+spl_srv_list = ['database-chassis', 'gbsyncd', 'e2scrub_reap']
```

Comment:
> Can you clarify what is e2scrub_reap?

@dgsudharsan it is debian internal service. According to the manual: `e2scrub attempts to check (but not repair) all metadata in a mounted ext[234] file system if the file system resides on an LVM logical volume. The block device of the LVM logical volume can also be passed in.` So we shouldn't take it into concideration.